keychain_1.10.go

```diff
@@ -85,13 +85,15 @@ var (
 		 kSecAttrAccount
 		 kSecAttrService
 	*/
-	SecClassGenericPassword SecClass = 1
+	SecClassGenericPassword  SecClass = 1
```

Comment:
Note for posterity: this diverges from keychain_pre_1.10.go, but I don't think that matters as we'll eventually drop that file.